<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="CBT-088" name="com.auvenir.ui.tests.functional.SmokeTest" tests="7" failures="1" timestamp="7 Mar 2017 11:39:46 GMT" time="432.436" errors="0">
  <testcase name="changingTheStatusAuditorToActive" time="32.923" classname="com.auvenir.ui.tests.functional.SmokeTest"/>
  <testcase name="clientLogsInAndOnBoards" time="75.050" classname="com.auvenir.ui.tests.functional.SmokeTest"/>
  <testcase name="auditorCreation" time="58.025" classname="com.auvenir.ui.tests.functional.SmokeTest"/>
  <testcase name="changingTheStatusAuditorToOnBoarding" time="58.528" classname="com.auvenir.ui.tests.functional.SmokeTest"/>
  <testcase name="adminLogin" time="32.202" classname="com.auvenir.ui.tests.functional.SmokeTest">
    <failure type="java.lang.AssertionError" message="Admin Login is not successfull expected [true] but found [false]">
        <![CDATA[java.lang.AssertionError: Admin Login is not successfull expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.auvenir.ui.tests.functional.SmokeTest.adminLogin(SmokeTest.java:143)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- adminLogin -->
  <testcase name="auditorOnboardingPage" time="58.654" classname="com.auvenir.ui.tests.functional.SmokeTest"/>
  <testcase name="invitingTheClient" time="117.054" classname="com.auvenir.ui.tests.functional.SmokeTest"/>
</testsuite> <!-- com.auvenir.functionaltestsmokeTest -->
